pkg/sharing/placeholder_secret_test.go

```diff
@@ -128,6 +129,15 @@ func Test_SecretReconciler_updatesStatus(t *testing.T) {
 
 		reload(t, &placeholderSecret, k8sClient)
 		assert.Equal(t, sourceSecret.Data[".dockerconfigjson"], placeholderSecret.Data[".dockerconfigjson"])
+		assert.Equal(t, sourceSecret.Data[".dockerconfigjson"], placeholderSecret.Data[".dockerconfigjson"])
+		fmt.Println("placeholder Annotations: ", placeholderSecret.ObjectMeta.Annotations)
+		fmt.Println("source Annotations: ", sourceSecret.ObjectMeta.Annotations)
+		for k, v := range sourceSecret.ObjectMeta.Annotations {
+			placeholderV, check := placeholderSecret.ObjectMeta.Annotations[k]
+			assert.True(t, check, fmt.Sprintf("failed to find source key %s in placeholder annotations", k))
+			assert.Equal(t, v, placeholderV)
```

Comment:
Why does `v` equal `placeholderV` here? Shouldn't the secrets have different annotations? Sorry if I am misinterpreting.

Reply:
right - sorry.  i was intentionally writing a failing test. - I meant for this to be more like, 'communication through source code' than 'a change proposal'.   I put it as a "draft PR" but should've been more explicit that I don't intend to merge. 

    if we wanted to merge something like this i would definitely have to make changes. :)

Reply:
Oh right. I should read the pr description :).